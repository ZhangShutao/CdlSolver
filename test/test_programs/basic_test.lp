p :- &c{s}.
q :- &c{-s}.