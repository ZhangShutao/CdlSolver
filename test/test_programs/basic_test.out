stable model: 1
X:a;
A:a;
Satisfiable
